src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -484,13 +561,14 @@ private Set<Long> mapInternalIdentifiersToConceptIdentifiers(Set<String> interna
 		return conceptIdentifiers;
 	}
 
-	private void removeIfFoundOnPath(List<Long> concepts, String path) {
+	private void removeFromCollectionIfFoundOnPath(Set<Long> concepts, String path) {
```

Comment:
Personally I prefer to return a list of concept ids found on the path and remove them in a separate method call.